README.md

```diff
@@ -1,25 +1,67 @@
-Lager graylog udp
-=================
+## lager_graylog
 
-There are two modules:
-* lager\_gelf\_formatter, which will format your logs according to gelf
-standards
-* graylog\_udp\_sender, which will send your logs to graylog instance
+This application provides lager backends for sending log messages to [Graylog](https://www.graylog.org/)
+over UDP or TCP, and the formatter module which spits out logs in [GELF](http://docs.graylog.org/en/stable/pages/gelf.html)
+format.
 
-Config
-======
+### TCP backend
 
+When you need a reliable log delivery, you can use the backend which connects to Graylog using TCP.
+This backend will also try to reconnect to Graylog indefinitely (with backoff) in case of connection
+failure.
+
+To use it, just declare it in your lager config:
+
+```erlang
+[{lager, [{handlers, [{lager_graylog_tcp_backend, [{host, "graylog-hostname"},
+                                                   {port, 12201}]
+
+}]}]}].
 ```
-[{lager, [
```

Comment:
I think it'd worth to show how to configure the `udp` backend.

Reply:
I'll "polish" the UDP backend in the next PR, including documentation. Is that okay or should I do it in this PR?

Reply:
That's fine.